startcorvus.sh

```diff
@@ -0,0 +1,8 @@
+killall -9 corvus
+
+/usr/bin/corvus -n 10.254.127.37:6379,10.254.127.48:6379,10.254.127.49:6379 -t 4 -b 6379 -L debug /etc/corvus.conf 1>/var/log/corvus.log 2>/var/log/corvus.err &
```

Comment:
Hello
I am a bit confused, as i didnt expect that init/startup scripts would appear in this merge request
However i have removed it
Should i restart pull request, or last commit will appear automatically in the request ?